tests/extrusion/test_facet_support_dofs.py

```diff
@@ -37,6 +37,13 @@ def hex_mesh():
                             2: [0, 2, 4, 5, 6, 7, 8, 9],
                             3: [1, 3, 4, 5, 6, 7, 10, 11]})])
 def test_hex(hex_mesh, args, kwargs, horiz_expected, vert_expected):
-    V = FunctionSpace(hex_mesh, *args, **kwargs)
+    if not kwargs:
+        fe = FiniteElement(args[0], hex_mesh.ufl_cell(), args[1], variant='equispaced')
+    else:
+        cells = hex_mesh.ufl_cell().sub_cells()
```

Comment:
`A, B = hex_mesh.ufl_cell().sub_cells()` (so that if the number of cells this cell contains changes it doesn't silently break)